tests/acquisition/covidcast/test_csv_importer.py

```diff
@@ -42,37 +42,38 @@ def test_is_sane_week(self):
     self.assertFalse(CsvImporter.is_sane_week(202054))
     self.assertFalse(CsvImporter.is_sane_week(20200418))
 
+
+  @patch("delphi.epidata.acquisition.covidcast.csv_importer.glob")
```

Comment:
question: why does this patch location need to be `delphi.epidata.acquisition.covidcast.csv_importer` when the patch for `isdir` on the line below can be in `os.path`?

Reply:
So the TLDR of this is due to Python imports
- `csv_importer` uses `import os`, so we can patch `os.path`, because `csv_importer` points to `os.path`
- `csv_importer` uses `from glob import glob`, which makes a new variable `glob` in the `csv_importer` namespace with the same value as `glob.glob`, but if we patch `glob.glob`, which updates that value to a Mock, `csv_importer.glob` retains its own value, so it's effectively not patched

So the end result is that you generally want to patch where a name is used, not a module itself.

[Here's a more complete blog post on this](https://nedbatchelder.com/blog/201908/why_your_mock_doesnt_work.html).